Documentation/CRDs/Shared-Filesystem/ceph-filesystem-crd.md

```diff
@@ -135,13 +135,14 @@ spec:
     # for more details see: https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers
     peers:
       secretNames:
-        - secondary-cluster-peer
+      # Add the secret name if it already exists else specify the empty list here.
     # specify the schedule(s) on which snapshots should be taken
     # see the official syntax here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#add-and-remove-schedules
     snapshotSchedules:
       - path: /
         interval: 24h # daily snapshots
-        startTime: 11:55
+        startTime: 2022-07-15T11:55:00
+        # The startTime should be in the format YYYY-MM-DDTHH:MM:SS
```

Comment:
The [ceph snapshot doc](https://docs.ceph.com/en/latest/cephfs/snap-schedule/#add-and-remove-schedules) shows `11:55` as an example time, just like we have in our doc. Including the date in the startTime just doesn't make logical sense if it's a recurring daily event. Why do we need the date if the ceph doc only shows a time?

Reply:
if we don't specify startTime , it takes the value as following `2022-07-19T00:00:00` for example : 
```
[root@centos-4gb-hel1-13 /]# ceph fs snap-schedule status |jq
{
  "fs": "myfs",
  "subvol": null,
  "path": "/",
  "rel_path": "/",
  "schedule": "3h",
  "retention": {},
  "start": "2022-07-19T00:00:00",
  "created": "2022-07-19T21:10:21",
  "first": null,
  "last": null,
  "last_pruned": null,
  "created_count": 0,
  "pruned_count": 0,
  "active": true
}
```
The above ceph documentation states : The start time is specified as a time string (more details about passing times below). By default the start time is last midnight.

....
When a timestamp is passed (the start argument in the add, remove, activate and deactivate subcommands) the ISO format %Y-%m-%dT%H:%M:%S will always be accepted. When either python3.7 or newer is used or https://github.com/movermeyer/backports.datetime_fromisoformat is installed, any valid ISO timestamp that is parsed by python’s datetime.fromisoformat is valid.
References : https://docs.ceph.com/en/latest/cephfs/snap-schedule/#usage

Reply:
i think we can skip it in the example spec and let it consider as default value further mentioning a comment in there that if a user wants to use `startTime` they can refer to the CephFS documentation if it sounds good ? And they should use the appropriate format regarding the same.

Reply:
I wonder if we could just handle this internally by prepending the date to the time in [AddSnapshotSchedule()](https://github.com/rook/rook/blob/ac06d035e6ef26c6fde3de576f004e3c0f73727a/pkg/daemon/ceph/client/filesystem_mirror.go#L98). Then it would be less confusing for users since they wouldn't need to worry about the date.

Reply:
@travisn do you mean prepending the current date in there in case the users only specify the time ?

Reply:
Yes, that was my thought. If the date is meaningless but required, the operator can prepend it if not specified by the user.

Reply:
Hi @travisn @BlaineEXE I have made the changes based on our discussion in the rook huddle meeting yesterday. Can you please review and share your thoughts if the changes look good ?

Reply:
As discussed further as a part of different PR we can split out a document for Mirroring under `Shared Filesystem (CephFS)` section : https://rook.io/docs/rook/latest/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-storage/